notebooks/1.data_splits/nbconverted/1.1_cell_injury_data_splits.py

```diff
@@ -0,0 +1,580 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Splitting feature selected cell injury profiles
+#
+# Here, we utilize the feature-selected cell-injury profiles generated in the preceding module notebook [here](../0.feature_selection/0.feature_selection.ipynb), focusing on dividing the data into training, testing, and holdout sets for machine learning training.
+#
+
+# In[1]:
+
+
+import json
+import pathlib
+import sys
+import warnings
+
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import train_test_split
+
+sys.path.append("../../")  # noqa
+from src.utils import (  # noqa
+    get_injury_treatment_info,
+    load_json_file,
+    split_meta_and_features,
+)
+
+# ignoring warnings
+warnings.catch_warnings(action="ignore")
+
+
+# ## Parameters
+#
+# Below are the parameters defined that are used in this notebook
+#
+
+# In[2]:
+
+
+# setting seed constants
+seed = 0
+np.random.seed(seed)
+
+# directory to get all the inputs for this notebook
+data_dir = pathlib.Path("../../data").resolve(strict=True)
+results_dir = pathlib.Path("../../results").resolve(strict=True)
+fs_dir = (results_dir / "0.feature_selection").resolve(strict=True)
+
+# directory to store all the output of this notebook
+data_split_dir = (results_dir / "1.data_splits").resolve()
+data_split_dir.mkdir(exist_ok=True)
+
+
+# In[3]:
+
+
+# loading feature selected cell injury dataset
+fs_profile_path = (fs_dir / "fs_cell_injury_only.csv.gz").resolve(strict=True)
+
+# load in feature select cell injury feature space file
+fs_feature_space = load_json_file(
+    fpath=(fs_dir / "fs_cell_injury_only_feature_space.json").resolve(strict=True)
+)
+fs_feats = fs_feature_space["features"]
+
+# load data
+fs_profile_df = pd.read_csv(fs_profile_path)
+
+
+# splitting meta and feature column names
+fs_meta, _ = split_meta_and_features(fs_profile_df)
```

Comment:
because you're splitting samples, shouldn't both fs and aligned data use the same splits?

Reply:
@gwaybio - I'm not quite sure what you mean by "same splits." The purpose of this function is to simply separate the metadata and morphological feature column names to make it easier for me to pass them into other functions. It doesn't actually perform any data splitting.

I may have misunderstood what you're trying to point out.

Reply:
oh gotcha, thanks for asking the clarification question!

Perhaps I am misunderstanding (or I have added this comment in an inappropriate place), but it seems like you are splitting train/test/holdout separately for the `fs` and `aligned` models - I recommend having one split (for both models) and the only difference is the feature space.